src/assets/scripts/client/ui/TutorialView.js

```diff
@@ -509,6 +567,28 @@ export default class TutorialView {
             side: 'left',
             position: tutorial_position
         });
+    */
+    }
+
+    /**
+     * Fetching descendent property of an object without using eval()
+     *
+     * Adapted from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#accessing_member_properties
+     *
+     * @for TutorialView
+     * @method _getPropFetcher
+     * @return [{function}] A function that will fetch a given descendent property of an object without using eval()
+     */
+    _getPropFetcher(desc) {
+        return ((obj) => {
+            let result = obj;
+            const arr = desc.split('.');
+            while (arr.length) {
+                if (result === undefined || result === null) return result;
```

Comment:
* please use block `if` statements
* you could use loose equality here to so we only need one check:
```js
if (result == null) { 
  
}
```

Reply:
done